catalog/core/catalog-core-camelcomponent/src/main/java/ddf/camel/component/catalog/content/FileSystemDataAccessObject.java

```diff
@@ -51,14 +52,15 @@ public void store(String storePath, String suffix, String key, Object value) {
                 }
             }
             file = new FileOutputStream(storePath + key + suffix);
-            OutputStream buffer = new BufferedOutputStream(file);
+            buffer = new BufferedOutputStream(file);
             output = new ObjectOutputStream(buffer);
             output.writeObject(value);
         } catch (IOException e) {
             LOGGER.debug("IOException storing value in cache with key = " + key, e);
         } finally {
```

Comment:
could we just move all of these to a try with resource and remove this finally block? Not required but I think we are trying to move away from finally for closing streams.

Reply:
Updated to use try with resource in each of these files.